docs/modules/ROOT/pages/governance.adoc

```diff
@@ -159,15 +161,15 @@ contract MyGovernor is Governor, GovernorCompatibilityBravo, GovernorVotes, Gove
         GovernorTimelockControl(_timelock)
     {}
 
-    function votingDelay() public pure override returns (uint256) {
-        return 6575; // 1 day
+    function votingDelay() public pure virtual override returns (uint256) {
```

Comment:
Is this really necessary? The wizard doesn't add virtual. I think it's better for the user if we remove unnecessary keywords.